tchannel/sync/client.py

```diff
@@ -20,18 +20,11 @@
 
 from __future__ import absolute_import
 
-from collections import namedtuple
-
-from concurrent.futures import TimeoutError
-from tornado import gen
-
-from tchannel import tornado as async
-from tchannel.tornado.hyperbahn import FIRST_ADVERTISE_TIME
-from tchannel.tornado.hyperbahn import AdvertiseError
+from tchannel import TChannel
 from threadloop import ThreadLoop
 
 
-class TChannelSyncClient(object):
+class TChannelSyncClient(TChannel):
```

Comment:
> Also, that let's us keep the current sync behavior unbroken until we start removing all this old code.

Do you mean you want to preserve `TChannelSyncClient` in addition to a `sync.TChannel` to make this backwards compatible?